code/game/objects/structures/window.dm

```diff
@@ -278,6 +283,20 @@
 	sleep(5) //Allow a littleanimating time
 	return TRUE
 
+/obj/structure/window/proc/hit_by_living(var/mob/living/M)
+	var/body_part = pick(BP_HEAD, BP_CHEST, BP_GROIN)
+	visible_message(SPAN_DANGER("[M] slams against \the [src]!"))
+	if(prob(30))
+		M.Weaken(1)
+	M.damage_through_armor(8, BRUTE, body_part, ARMOR_MELEE)
+
+	var/tforce = 15
+	if(reinf) tforce *= 0.25
+	if(health - tforce <= 7 && !reinf)
```

Comment:
is this enough to break a basic window in one jump?

Reply:
No. It does the same amount of damage an aggressive grab banging does.

Reply:
> Cooldown is bad game design, but nutrition... That'd probably even make the gamers go to bar. A brilliant idea

One thing I'm not sure about is FBPs. Aren't they immune to nutrition?
(Probably a separate PR would be needed, to make FBPs require electricity for nutrition - like, draining batteries or feasting at APCs to regenerate it).